src/CONST.ts

```diff
@@ -1798,6 +1799,7 @@ const CONST = {
                 // Here we will add other connections names when we add support for them
                 QBO: 'quickbooksOnline',
                 XERO: 'xero',
+                NETSUITE: 'netsuite',
```

Comment:
question, I was confused if we should use `NETSUITE` as single word or `NET_SUITE` underscore separated.

Reply:
one word should be fine I believe